SecretDecks® Newsletter to Occultish.md

```diff
@@ -1,15 +1,18 @@
-## Intro
-Calling all creators! SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
-Are you an artist? Build a new global fanbase, make a living off your digital art without any middlemen, as well as own & manage your copyrights!
-What if you're not an aritst? Join us anyway and find out how you can help us to help each other and what you can get in return! 
+## Calling all creators! 
+SecretDecks® looks to you if you’ve ever reinterpreted the Tarot and made your own deck... or perhaps you feel inspired to do so? 
+## Are you an artist?  
+Build a new global fanbase, make a living off your digital art without any middlemen, as well as own & manage your copyrights!
```

Comment:
[...] without any middlemen **and through the medium of your choice**, as well as own [...]

Regarding your comment @wlstylr , maybe sth like the above?

Reply:
Yes, this is good.  Sorry Dan, I didn't see until now.

Reply:
All good Tyler, will merge 

> > looking vry nice, guys! I only have a few small things like taking out a comma towards the end of line 15 and replacing an ampersands with "and". Is the best way for something like this to wait until it's merged to the main and then directly edit?
> 
> I agree with all suggestions. Would it make sense for me to merge this original request and then the suggestions be added by those that suggested through separate pull requests? @DanM3rcurius @stephen-rowan

Very good question (I have to add it to the GitHub FAQ!): 
Until the changes from a PR have not been merged into the main, all edits you do in your repo, you can integrate into the same PR.  Think I'll wait for your edits and then merge? 
For larger feedback / edits (eg whole paragraphs,...) it might be better to merge the PR first and then open separate issue. I'd love to start (or continue?!) GitHub Workshops/Tutorials to identify the workflows currently best suited for us. 

@stephen-rowan What do you think? Will the contribution data trail be the same if edits are merged into the same PR?

Reply:
I updated my  repo with the suggested edits from @DanM3rcurius and @evolayak.  It looks like I may have created some problems, as now resolving conflicts and only be done via command line.  I second your idea of having GitHub Workshops. 🙏

Reply:
Response to https://github.com/SecretDecks/Contentwriting/pull/30#discussion_r656382066

Maybe have a Zoom this week to try to sort it out on screen share ?

Reply:
Yes @stephen-rowan, that would be helpful.  Thank you :)